fast_api/routes/organization.py

```diff
@@ -56,11 +66,47 @@ def get_user_info(request: Request):
     return {"data": {"userInfo": data}}
 
 
+@router.get("/get-user-info-extended")
+def get_user_info_extended(request: Request):
+    user_id = str(auth_manager.get_user_by_info(request.state.info).id)
+    user = get(user_id)
+    name, mail = resolve_user_name_and_email_by_id(user_id)
+
+    data = {
+        "userInfo": {
+            "id": user_id,
+            "organizationId": str(user.organization_id),
+            "firstName": name.get("first"),
+            "lastName": name.get("last"),
+            "mail": mail,
+            "role": user.role,
+            "languageDisplay": user.language_display,
+        }
+    }
+
+    return pack_json_result({"data": data})
+
+
+@router.get("/get-user-info-mini")
+def get_user_info_mini(request: Request):
+    user_id = str(auth_manager.get_user_by_info(request.state.info).id)
+    user = get(user_id)
```

Comment:
we first get the user to extract the id to then get the user?
(the object returned by get_user_by_info is the one from the db already)

```suggestion
    user = auth_manager.get_user_by_info(request.state.info)
    user_id = str(user.id)
```

same for above